app/views/content/steps-to-become-a-teacher.md

```diff
@@ -81,6 +84,19 @@
     - Advisor
     - Teacher Training Advisor
     - get an adviser
+  inset_text:
+    international-content:
+      text: There are more steps to consider if you're <a href="/non-uk-teachers">a non-UK citizen</a>.
+      color: grey
+---
+<b>Discover if a career teaching in a primary or secondary school in England is right for you.</b>
```

Comment:
Is `<b>` the right formatting to use here rather than `<strong>`? Maybe a question for @ethax-ross?